README.rst

```diff
@@ -33,7 +33,7 @@ a two-qubit circuit.
 
     circuit = Circuit(2)
 
-    with circuit.context as q:
+    with circuit.context as (q, c):
```

Comment:
That's a quite nice idea actually! I'll think a bit about it (but will probably not make the change in this PR).

Reply:
I want to second this idea. It will also make future changes easier because you won't need to, for example, update all of your test code like you did in this PR if you add another field. So it ends up keeping future changes more "contained".

Reply:
Fixed in #13.